### PR TITLE
CSS class is now added also to the menu items with children

### PR DESCRIPTION
**Pull request by enriconardo**
I noticed that the menu items with children are not printed with the CSS classes. I added them.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ vendor/
 composer.lock
 Modules/
 build/
+nbproject/
```
Comment (nWidart) [inline .gitignore]: This seems to be something related to your environment,  please add this to your global .gitignore file instead.

**Comment by nWidart** [comment]
Thank you for the change. 👍  Please also make sure you don't leave any whitespaces at end of lines.

**Comment by nWidart** [comment]
Oh wait, this is for 2.0. This shouldn't be sent here.
Please make your change on the Platform repository, 2.0 branch instead. The documentation has these steps: https://asgardcms.com/docs/v2/getting-started/contributing#getting-setup

**Comment by enriconardo** [comment]
Sorry I'm not yet good with pull request

**Pull request closed without merging**